sql/hive-thriftserver/pom.xml

```diff
@@ -148,16 +148,6 @@
       <artifactId>byte-buddy-agent</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
Is it possible for us to add some configuration in SparkBuild.scala to ensure that `hive-llap` is not included in the classpath too when testing the thriftserver module with sbt?

Reply:
Done by updating the `SparkBuild.scala`, it could be verified by
```
build/sbt -Phive-thriftserver hive-thriftserver/Test/dependencyTree | grep hive-llap
```

before
```
[info]   +-org.apache.hive:hive-llap-client:2.3.10
[info]   +-org.apache.hive:hive-llap-common:2.3.10
[info]   | +-org.apache.hive:hive-llap-client:2.3.10
[info]   | +-org.apache.hive:hive-llap-common:2.3.10
```

now result is empty.